modules/morphology/src/main/resources/schemas/neurosciencegraph/morphology/swctoascconvertion/v0.1.0.json

```diff
@@ -0,0 +1,47 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/morphology/reconstruction/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0",
```

Comment:
I suggest we don't have a schema for every data type to data type conversion (e.g swc to asc).